### ISSUE TITLE
Design Intent - Accessibility Feedback - Ask VA, Business vs. Personal Flows Decoupling (authenticated only)

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#138342]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._

- [x] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [x] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [x] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [x] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Thoughts/questions

- 

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Should:** The cards with action links should be in a list structure, so that a screen reader user can built context around there being two choices - the screen reader will announce the list and the number of list items, which will provide some context. While this is not as strong as a radio tile, it at least builds some context through semantics. I would encourage your team to reconsider the radio tile, as communication and interaction are "stronger" than the cards with action links. 
- [x] **Must:** UI elements must meet a contrast ratio of 3:1, so that they are perceivable by people with low vision and other vision impairments. The current progress bar only has a ratio of 2.4:1 (orange to gray). Consider updating to the VADS styling for progress bars. 

<img width="473" height="678" alt="progress bar contrast" src="https://github.com/user-attachments/assets/d6d1d7da-5498-4405-9950-01d6231f00c7" />

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app
